src/Microsoft.Azure.WebJobs.Host/Bindings/AmbientBindingContext.cs

```diff
@@ -10,7 +10,7 @@ namespace Microsoft.Azure.WebJobs.Host.Bindings
     /// <summary>
     /// Context for runtime bindings performed via <see cref="IBinder"/>.
     /// </summary>
-    public class AmbientBindingContext
+    internal class AmbientBindingContext
```

Comment:
This was made public when I added IBinderEx. I'm removing IBinderEx in this PR, so this reverts back